pulpcore/cli/ansible/repository.py

```diff
@@ -211,3 +222,21 @@ def sync(
         href=repository_href,
         body=body,
     )
+
+@repository.command()
+@name_option
+@href_option
+@click.option("--signing-service", required=True, callback=_signing_service_callback)
+@click.option("--content-units", callback=load_json_callback)
```

Comment:
Is there a better way to add these options? We have so many helper functions that I'm starting to forget what they can do...

Reply:
I think, they are just fine here.